### PR TITLE
amp ad preview design

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@aghassemi PTAL

Demo: https://amp-by-example-staging.appspot.com/amp-ads/experimental_ads/scrollbound_video_ad/preview/

File: templates/preview-a4a.html [hunk 1]
```diff
@@ -5,95 +5,162 @@
...
+      {{/metadata.scrollToReveal}}
+      <p class="pb1 xs-hide sm-hide">Lorem ipsum dolor sit amet, etiam mattis ullamcorper lacinia integer et, nec aliquam libero, vel duis pretium hendrerit eget lacus, nullam massa rhoncus vel congue luctus porta. Amet cursus integer lacus mi pede, turpis dictumst nunc dolor donec tortor, enim velit elementum, mollis elit purus vestibulum eget amet ipsum. Ipsum placerat neque voluptatem arcu in integer, dolor sem quisque, mattis nulla massa ipsum enim cursus ut, etiam per adipiscing mauris tellus tortor, sapien maecenas pharetra amet ut felis. </p>
       <div class="abe-amp-ad-container">
         {{#metadata.adSlot}}
-        <amp-ad width="{{width}}" height="{{height}}"
-           type="doubleclick"
-           data-slot="{{metadata.adSlot}}">
+        <amp-ad width="{{width}}" 
+                height="{{height}}"
+                type="doubleclick"
```
Comment (aghassemi) [inline templates/preview-a4a.html]: we can add layout="{{layout}}" here too

**Comment by sebastianbenz** [comment]
All done. PTAL

**Comment by aghassemi** [comment]
LGTM

**Pull request merged**